### ISSUE TITLE
FALCOM CLASSICS2                                                                                                

### ISSUE THREAD

**Issue by devmiyax** [devmiyax/yabause#187]
version:0.2.3
device:502SH
cpu_type:2
video_type:1
????????????
![screenshot](http://www.uoyabause.org//system/reports/screenshots/000/000/772/original/_storage_emulated_0_yabause_screenshots_T-31505G_2016_03_03_08_21_16.png?1456961141)

**Comment by devmiyax** [comment]
Infinity Loop of CD command

```
yabause\src\cs2.c:1068): cs2	: Command: setCDDeviceConnection 0f96 3000 0000 0000 0000

yabause\src\cs2.c:1056): cs2	: Command: playDisc 0fd6 1082 a1b0 0080 000b

yabause\src\cs2.c:879): partition number = 0 blocks = 65 blockfreespace = 135 fad = 2a1b1 playpartition->size = 20800 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 66 blockfreespace = 134 fad = 2a1b2 playpartition->size = 21000 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 67 blockfreespace = 133 fad = 2a1b3 playpartition->size = 21800 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 68 blockfreespace = 132 fad = 2a1b4 playpartition->size = 22000 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 69 blockfreespace = 131 fad = 2a1b5 playpartition->size = 22800 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 70 blockfreespace = 130 fad = 2a1b6 playpartition->size = 23000 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 71 blockfreespace = 129 fad = 2a1b7 playpartition->size = 23800 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 72 blockfreespace = 128 fad = 2a1b8 playpartition->size = 24000 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 73 blockfreespace = 127 fad = 2a1b9 playpartition->size = 24800 isbufferfull = 0

yabause\src\cs2.c:1146): cs2	: Command: getThenDeleteSectorData 0f56 6300 0000 0000 0001

yabause\src\cs2.c:1148): cs2	: ret: 0f57 0300 4101 0102 a1b9

yabause\src\cs2.c:1051): cs2	: Command: endDataTransfer 0f54 0600 0000 0000 0000

yabause\src\cs2.c:1053): cs2	: ret: 0fd5 0300 0400 0000 0000

yabause\src\cs2.c:879): partition number = 0 blocks = 73 blockfreespace = 127 fad = 2a1ba playpartition->size = 24800 isbufferfull = 0

yabause\src\cs2.c:879): partition number = 0 blocks = 74 blockfreespace = 126 fad = 2a1bb playpartition->size = 25000 isbufferfull = 0

yabause\src\cs2.c:895): PLAY HAS ENDED

yabause\src\cs2.c:1146): cs2	: Command: getThenDeleteSectorData 0f56 6300 0000 0000 0001

yabause\src\cs2.c:1148): cs2	: ret: 0f57 0100 4101 0102 a1bb

yabause\src\cs2.c:1051): cs2	: Command: endDataTransfer 0f54 0600 0000 0000 0000

yabause\src\cs2.c:1053): cs2	: ret: 0fd5 0100 0400 0000 0000

yabause\src\cs2.c:1146): cs2	: Command: getThenDeleteSectorData 0f56 6300 0000 0000 0001

yabause\src\cs2.c:1148): cs2	: ret: 0f57 0100 4101 0102 a1bb

yabause\src\cs2.c:1051): cs2	: Command: endDataTransfer 0f54 0600 0000 0000 0000

yabause\src\cs2.c:1053): cs2	: ret: 0fd5 0100 0400 0000 0000

yabause\src\cs2.c:1146): cs2	: Command: getThenDeleteSectorData 0f56 6300 0000 0000 0001

yabause\src\cs2.c:1148): cs2	: ret: 0f57 0100 4101 0102 a1bb

yabause\src\cs2.c:1051): cs2	: Command: endDataTransfer 0f54 0600 0000 0000 0000

yabause\src\cs2.c:1053): cs2	: ret: 0fd5 0100 0400 0000 0000

yabause\src\cs2.c:1146): cs2	: Command: getThenDeleteSectorData 0f56 6300 0000 0000 0001

yabause\src\cs2.c:1148): cs2	: ret: 0f57 0100 4101 0102 a1bb

yabause\src\cs2.c:1051): cs2	: Command: endDataTransfer 0f54 0600 0000 0000 0000

yabause\src\cs2.c:1053): cs2	: ret: 0fd5 0100 0400 0000 0000

yabause\src\cs2.c:1146): cs2	: Command: getThenDeleteSectorData 0f56 6300 0000 0000 0001

yabause\src\cs2.c:1148): cs2	: ret: 0f57 0100 4101 0102 a1bb

yabause\src\cs2.c:1051): cs2	: Command: endDataTransfer 0f54 0600 0000 0000 0000

yabause\src\cs2.c:1053): cs2	: ret: 0fd5 0100 0400 0000 0000

yabause\src\cs2.c:1146): cs2	: Command: getThenDeleteSectorData 0f56 6300 0000 0000 0001

yabause\src\cs2.c:1148): cs2	: ret: 0f57 0100 4101 0102 a1bb

yabause\src\cs2.c:1051): cs2	: Command: endDataTransfer 0f54 0600 0000 0000 0000

yabause\src\cs2.c:1053): cs2	: ret: 0fd5 0100 0400 0000 0000
```

**Comment by devmiyax** [comment]
MSLC(Ｗ) ; MonitorSLot Call 
???

**Comment by devmiyax** [comment]
Sound seems not cause of problem.
The problem is start button is not respond.

**Comment by devmiyax** [comment]
Read INTBACK data at PC 0x06011628

**Comment by devmiyax** [comment]
fixed by de9cb8ed9670e4f977393eb8244f5c632508c615

**Comment by devmiyax** [comment]
![image](https://cloud.githubusercontent.com/assets/1174152/15228749/aee94ed2-18c8-11e6-986f-85800b5fb7a5.png)
![image](https://cloud.githubusercontent.com/assets/1174152/15228759/bca4c920-18c8-11e6-83e4-1b82e642fb44.png)
